packages/app/src/utils/networks.ts

```diff
@@ -216,4 +235,6 @@ export const NETWORK_MULTI_SEND_CONTRACT: Record<NETWORK, string> = {
   [NETWORK.LINEA_SEPOLIA]: "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761",
   [NETWORK.PLASMA_TESTNET]: "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761",
   [NETWORK.PLASMA]: "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761",
+  [NETWORK.ZETACHAIN_TESTNET]: "",
```

Comment:
```suggestion
  [NETWORK.ZETACHAIN_TESTNET]: "0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761",
```
You can add the same address for testnet. I just deployed it